### PR TITLE
Tweak header logo sizes

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Header census logo changes
Also:
- renames xs breakpoint to xxs and xs breakpoint is at new width and updates components to work with the new breakpoints
- tidies some css
- adds new smaller nisra logo for smaller breakpoints

### How to review 
Check census header logo looks correct at all view width sizes and conforms with non census headers

### CODE REVIEW

**Comment by rmccar** [comment]
[Adjust census logo size at breakpoints](https://trello.com/c/zgSU2Y4Y/850-adjust-census-logo-size-at-breakpoints)

**Comment by jrbarnes9** [comment]
Think we need a `min-height` or something on the header, so it doesn't change height between button/no button

Screenshot 2021-01-08 at 13 09 31

**Review by boxadesign — DISMISSED** [review]

**Comment by jrbarnes9** [comment]
Need to reduce size of NISRA logo on mobile
Screenshot 2021-01-08 at 17 07 22

**Comment by jrbarnes9** [comment]
Census logo isn't left aligned

Screenshot_20210108-213251

**Comment by jrbarnes9** [comment]
The height of the blue header banner still different between these two examples...

Screenshot 2021-01-11 at 13 51 15

File: src/components/header/_header.scss [hunk 1]
```diff
@@ -12,7 +12,8 @@
     height: auto;
     fill: $color-white;
 
-    @include mq(xs, s) {
+    @include mq(xs, 399px) {
+      padding: 0.45rem 0;
       max-width: 145px;
```
Comment (jrbarnes9) [inline src/components/header/_header.scss:17]: Could we increase this slightly, to 160px?  Do we need the padding?

**Comment by jrbarnes9** [comment]
The height of the `header__top` should change at the same breakpoint the ons logo switches from stacked to single line (at 399px not 500px)

File: src/components/header/_header.scss (hunk 1 above)
Comment (rmccar) [reply src/components/header/_header.scss]: The padding is to make up the difference between the svg logo used in the census style and the text used in the regular style. Another option would be to try look at font sizes compared to the svg size and try to match the size of them including any whitespace in the svgs

File: src/scss/vars/_grid.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 $grid-bp: (
   xs: 300px,
+  xss: 400px,
```
Comment (jrbarnes9) [inline src/scss/vars/_grid.scss:3]:
```
xxs: 300px
xs: 400px
```
...would this have a negative impact apart from a find and replace job in the DS? Just think Medium, Small, Extra Small, Extra extra small - is a more logical sequence.

**Comment by boxadesign** [comment]
Spacing looks off between the title and tagline.

**Comment by boxadesign** [comment]
This may have been a design decision but at this breakpoint there's either too much padding or the site title is too small... (@jrbarnes9 )


image

**Review by jrbarnes9 — COMMENTED** [review]
1. Still different heights of header banner - min height needs to match height of header with button.
Screenshot 2021-01-20 at 11 23 31

2.  Can we reduce the height of the header banner at 740 (when 'save' btn is moved to footer, and menu is collapsed to btn)?  Currently doesn't reduce til 400px. So unnecessarily tall.
Screenshot 2021-01-20 at 11 23 17

3. Header with description is not vertically centre-aligned...
Screenshot 2021-01-20 at 11 22 31

**Review by jrbarnes9 — COMMENTED** [review]
Title with description still not vertically centred on desktop

Screenshot 2021-01-26 at 12 35 26

**Review by jrbarnes9 — APPROVED** [review]

**Pull request merged**